driving_gridworld/road.py

```diff
@@ -215,12 +217,14 @@ def successors(self, action):
                     num_avail_spaces_given_revealed_obs = (
                         len(self._available_spaces) - num_obstacles_revealed)
                     prob *= p / float(num_avail_spaces_given_revealed_obs)
-                    num_obstacles_revealed += 1
                 else:
                     next_obstacle = obs.next(self._car)
                     prob *= 1.0 - p
                 next_obstacles.append(next_obstacle)
                 reward += next_obstacle.reward(self._car)
+            reward += self._car.reward()
+            if self._car.col == 0 or self._car.col == 3:
+                reward += -4 * self._car.speed
```

Comment:
Please change this to use `-=`.

Reply:
Hi Dustin, you meant just to change line 227, right? Not 225 as well?

Reply:
Correct; changing 225 would just make the code more complicated because it would have to change to `reward -= -self._car.reward()`.